pkg/transform/reportoutput/staticpage/js/custom.js

```diff
@@ -33,3 +40,82 @@ function fillEmptyTableData() {
     }
   });
 }
+
+function sortDateRows() {
+  commonTableSort('.date-th',
+    function (a, b) {
+      var aDate = $(a).children(".date-td").text();
+      var bDate = $(b).children(".date-td").text();
+
+      return new Date(aDate) - new Date(bDate);
+    },
+    function (a, b) {
+      var aDate = $(a).children(".date-td").text();
+      var bDate = $(b).children(".date-td").text();
+
+      return new Date(bDate) - new Date(aDate);
+    });
+}
+
+function sortStringRows() {
+  commonTableSort('.string-th',
+    function (a, b) {
+      var aStr = $(a).children('.string-td').text();
+      var bStr = $(b).children('.string-td').text();
+
+      return aStr.localeCompare(bStr);
+    },
+    function (a, b) {
+      var aStr = $(a).children('.string-td').text();
+      var bStr = $(b).children('.string-td').text();
+
+      return bStr.localeCompare(aStr);
+    });
+}
+
+function commonTableSort(th, ascSortFunc, descSortFunc) {
+  // set initial arrow icon
+  $(th).append(iconDown);
+  $(th).children('svg').css("margin-left", "10px");
+
+  $(th).click(function () {
+    var parentTable = $(this).parents()[2];
+    var parentTableHead = $(parentTable).children('thead');
+    var tableHeaders = $($(parentTableHead).children('tr')).children('th');
+
+    var parentTableBody = $(parentTable).children('tbody');
+    var tableRows = $(parentTableBody).children('tr').toArray();
+
+    var sorted = $.parseJSON($(this).attr('sorted'));
+
+    if (!sorted) {
```

Comment:
Avoid code duplication here might be overkill but do we need the same comment twice?

Reply:
Right, I thought this I removed it.